### PR TITLE
Adding SMSLink Connector to Microsoft Power Automate, Power Apps, and Azure Logic Apps connectors (master)

### PR DESCRIPTION
**Pull request by smslink**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

File: certified-connectors/SMSLink/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SMSLink",
+    "description": "SMSLink is a SMS provider that provides integrated services such as SMS Marketing, SMS Gateway and Mail to SMS.",
+    "version": "1.0"
+  },
```
Comment (sriyen-msft) [inline certified-connectors/SMSLink/apiDefinition.swagger.json:7]: @smslink Please add "contact" section to info

File: certified-connectors/SMSLink/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      "name": "connection_id"
+    }
+  },
+  "security": [
+    {
+      "API Key": []
+    }
+  ],
+  "tags": []
+}
```
Comment (sriyen-msft) [inline certified-connectors/SMSLink/apiDefinition.swagger.json:89]: @smslink Please add "x-ms-connector-metadata" section

File: certified-connectors/SMSLink/apiDefinition.swagger.json (hunk 1 above)
Comment (smslink) [reply certified-connectors/SMSLink/apiDefinition.swagger.json:11]: Done. Please note that contact section is not well documented, most connectors don't have this section, so we had to search one connector and use the syntax from there.

File: certified-connectors/SMSLink/apiDefinition.swagger.json (hunk 2 above)
Comment (smslink) [reply certified-connectors/SMSLink/apiDefinition.swagger.json]: Done.

**Comment by smslink** [comment]
@sriyen-msft We have resolved the 2 requested changes (contact and x-ms-connector-metadata).

**Pull request merged**